examples/js/loaders/GLTFLoader.js

```diff
@@ -2242,13 +2242,13 @@ THREE.GLTFLoader = ( function () {
 
 		// Nothing in the node definition indicates whether it is a Bone or an
 		// Object3D. Use the skins' joint references to mark bones.
-		for ( var skinIndex in skins ) {
+		for ( var i =  0; i < skins.length; i ++ ) {
```

Comment:
Could we keep the variable name as `skinIndex` here, and `nodeIndex` below? Given the multiple and nested `for` loops in this method, I think those names are more readable.

Reply:
That's no problem.